### ISSUE TITLE
Login section UI

### ISSUE THREAD

**Issue by Ardianpalloshi** [amosproj/amos2022ws05-shared-desk-mgmt#9]
## User story

1. As a user
2. I want / need to see a login section
3. So that I can enter my login data

## Acceptance criteria

- Login section appears after hitting the login button
- Login section provides fields to enter the login data - email+password
- Login section contains a button to start the login function
- Login section should show an error message to the user if the login fails and disappear if login was successful

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release
